tests/inventory.yaml.j2

```diff
@@ -1 +1,13 @@
-{{ { 'all': { 'hosts': hostvars } } | to_yaml }}
+{% set role_vars = vars.keys() | intersect((
+     lookup('file', '../defaults/main.yml', errors='ignore') | from_yaml
+   ).keys())
+-%}
+{{ {
+  'all': {
+    'hosts': hostvars | to_json | from_json | combine(dict(
```

Comment:
`to_json | from_json` ?

Reply:
The explanation of this construct is in [commit](https://github.com/linux-system-roles/timesync/pull/72/commits/7c4d07db2a65c8a7d41943ad070a3fe637366c89) message:
> The `hostvars | to_json | from_json` construct aims to convert HostVars
and HostVarsVars instances to Python dictionaries so `combine` filter
will apply correctly (note that `dict(hostvars)` is just not enough
because it only makes a shallow conversion to dict which ends up with
`hostvars[host]` will be totally ignored by `combine`).

In short, I need to convert `hostvars` to `dict` deeply, but I not found the better solution then `to_json | from_json` yet.

Reply:
What do you mean by "Python dictionaries" in Jinja2 context?

Reply:
I Jinja2 context, I mean objects that are instances of `collections.abc.MutableMapping`.